src/libcollectionstest/vec.rs

```diff
@@ -213,6 +213,16 @@ fn test_retain() {
 }
 
 #[test]
+fn test_retain_mut() {
```

Comment:
I don't believe this test is necessary since as far as I known the examples in doc comments are already executed as if they were tests.